Services/LocalizationSKUUniquenessExceptionProvider.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Nwazet.Commerce.Models;
+using Orchard.ContentManagement;
+using Orchard.Environment.Extensions;
+using Orchard.Localization.Models;
+using Orchard.Localization.Services;
+
+namespace Nwazet.Commerce.Services {
+    [OrchardFeature("Nwazet.AdvancedSKUManagement")]
+    public class LocalizationSKUUniquenessExceptionProvider : ISKUUniquenessExceptionProvider {
```

Comment:
Isn't that a helper more than an exception provider, since it doesn't seem to provide any exceptions?